examples/echoserver/README_wolfsentry.md

```diff
@@ -0,0 +1,276 @@
+# wolfSentry Itegration Example
+
+This is a basic example that uses wolfsentry with SSH server. wolfSentry will filter a ssh client when it types invalid password more than a threshold value.
+Once the peer breaks the theshold number, the peer is accommodated into a penalty box with the certain penalty time.
+
+# Prerequisites
+
+This example is designed to be run an environment that has file system because JSON configuration file is expected to be read.
+
+# Build echoserver with wolfsentry
+
+## Build and Install wolfsentry
+Build and Install wolfSentry
+
+```
+$ cd <wolfsentry root folder>
+$ make -j check
+$ sudo make install
+```
+## Build wolfSSH
+
+Build wolfSSH with the following configuration option
+
+```
+$ cd <wolfSSH root folder>
+$./configure --enable-wolfsentry`
+$ make
+```
+
+# wolfSentry Configuration
+
+## Filtering IP Address
+The example is configured by JSON configuraiotn file to only accept connetions from 127.0.0.1 or 192.168.11.3. Change `192.168.11.3` to your preference IP address first.
```

Comment:
Spelling - "connetions" should be "connections".  I also think "preference" should be "preferred".

Reply:
thanks. fixed